packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -36,6 +37,7 @@ export function RadioButtonGroup<T>({
   className,
   fullWidth = false,
   autoFocus = false,
+  ariaLabel
```

Comment:
```suggestion
  ariaLabel,
```
Forgot to address this. A good reminder to use Prettier!

Reply:
@khushijain21 Let's make the CI happy :)

Reply:
still shows error, can you help?

Reply:
Ah, yes, they have updated so now it should be `aria-label` instead.